src/main/java/org/n52/challenge/OwmApplication.java

```diff
@@ -9,11 +11,23 @@
  * @author Matthes Rieke <m.rieke@52north.org>
  */
 public class OwmApplication {
-    
+
     private static final Logger LOG = LoggerFactory.getLogger(OwmApplication.class);
-    
-    public static void main(String[] args) {
-        LOG.info("{} started", OwmApplication.class.getSimpleName());
+
+    public static void main(String[] args){
+
+        try {
+            
+            WeatherController controller = new WeatherController();
+            CityWeather city = controller.retrieveWeather("Muenster");
+            LOG.info("Current weather in Münster:{}", city.getData());
+            LOG.info("Weather as JSON:\n {}", controller.deseralize(city));
+            
+        } catch (APIException | JsonProcessingException ex) {
+            java.util.logging.Logger.getLogger(OwmApplication.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
For logging the exception, you should have used the `LOG` object as for the other output.